File: sub-web-model/src/main/java/org/github/base/service/impl/TabSafeQuestionTypeServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.TabSafeQuestionTypeEntity;
import org.github.base.mapper.ITabSafeQuestionTypeMapper;
import org.github.base.service.ITabSafeQuestionTypeService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 安全问题类别 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-14
 */
@Service
public class TabSafeQuestionTypeServiceImpl extends ServiceImpl<ITabSafeQuestionTypeMapper, TabSafeQuestionTypeEntity> implements ITabSafeQuestionTypeService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/TabSafeQuestionTypeServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.TabSafeQuestionTypeEntity;
import org.github.mysql.mydb.base.mapper.ITabSafeQuestionTypeMapper;
import org.github.mysql.mydb.base.service.ITabSafeQuestionTypeService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 安全问题类别表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class TabSafeQuestionTypeServiceImpl : ServiceImpl<ITabSafeQuestionTypeMapper, TabSafeQuestionTypeEntity>(), ITabSafeQuestionTypeService
